app/queries/internal/moderators_query.rb

```diff
@@ -0,0 +1,36 @@
+module Internal
+  class ModeratorsQuery
```

Comment:
Because this is internal I think this will work fine for now but we might want to put a note or comment here that these lookups can be done VERY fast in Elasticsearch rather than having to burden Postgres with the lookup. The benefit to that is we already have all those query classes setup, all you would need to do is use them!

Reply:
I agree 100% with using Elasticsearch on that. I'm gonna take a look later today, but maybe into another PR to be easy to review, without all the UI changes. WDYT?

Reply:
Totally fine doing it in a separate PR since this is already tested and ready to go!